src/component/user/common/ResetPasswordForm/PasswordChecker/PasswordChecker.tsx

```diff
@@ -35,6 +35,8 @@ const LOWERCASE_ERROR =
     'The password must contain at least one lowercase letter.';
 const REPEATING_CHARACTER_ERROR =
     'The password may not contain sequences of three or more repeated characters.';
+export const PASSWORD_FORMAT_MESSAGE =
+    'The password must be at least 10 characters long, and include an uppercase letter, a lowercase letter, a number, and a symbol.';
```

Comment:
Just a minor text adjustment suggestion, this. Update at your discretion ☺️ 

```suggestion
    'The password must be at least 10 characters long and must include an uppercase letter, a lowercase letter, a number, and a symbol.';
```